exchanges/kraken/kraken_websocket.go

```diff
@@ -1127,18 +1139,14 @@ func (k *Kraken) GenerateDefaultSubscriptions() ([]stream.ChannelSubscription, e
 			})
 		}
 	}
-	return subscriptions, nil
-}
-
-// GenerateAuthenticatedSubscriptions Adds default subscriptions to websocket to be handled by ManageSubscriptions()
-func (k *Kraken) GenerateAuthenticatedSubscriptions() ([]stream.ChannelSubscription, error) {
-	var subscriptions []stream.ChannelSubscription
-	for i := range authenticatedChannels {
-		params := make(map[string]interface{})
-		subscriptions = append(subscriptions, stream.ChannelSubscription{
-			Channel: authenticatedChannels[i],
-			Params:  params,
-		})
+	if k.Websocket.CanUseAuthenticatedEndpoints() {
+		for i := range authenticatedChannels {
+			params := make(map[string]interface{})
+			subscriptions = append(subscriptions, stream.ChannelSubscription{
+				Channel: authenticatedChannels[i],
+				Params:  params,
```

Comment:
```go
Params:  make(map[string]interface{}),
```
🤷  or just rm the field if you can?

Reply:
Really not sure why that was there, but good catch that I couldn't just copy paste :D